src/HttpHeader.cc

```diff
@@ -356,15 +355,15 @@ HttpHeader::parse(const char *buf, size_t buf_len, bool atEnd, size_t &hdr_sz)
         blk_end = blk_start + strlen(blk_start);
     }
 
-    if (parse(blk_start, blk_end - blk_start)) {
+    if (parse(blk_start, blk_end - blk_start, interpreter)) {
         hdr_sz = parse_start - buf;
         return 1;
     }
     return -1;
 }
 
 int
-HttpHeader::parse(const char *header_start, size_t hdrLen)
+HttpHeader::parse(const char *header_start, size_t hdrLen, Http::ContentLengthInterpreter &interpreter)
```

Comment:
Please name the new parameter `clen` in this method implementation. It will reduce change noise and make method code slightly more readable.